database_admin/schema/create_schema.sql

```diff
@@ -659,6 +659,7 @@ CREATE TABLE IF NOT EXISTS system_platform
     third_party              BOOLEAN                  NOT NULL DEFAULT false,
     baseline_id              INT,
     baseline_uptodate        BOOLEAN,
+    yum_updates              TEXT,
```

Comment:
You need to update schema migrations as well. I think test fail due to that. Check commit: "SPM-736: store thirdparty system flag".

Reply:
That was it. Thanks!

Reply:
:+1:

Reply:
This column will store json data so it should be `JSONB` type.